### PR TITLE
adding option to ignore files with unwanted extentions

### PR DESCRIPTION
**Pull request by sergey-stratoscale**
@Stratoscale/systegration PR
due to log rotation, ignoring .gz files is needed

### CODE REVIEW

**Comment by rom-stratoscale** [comment]
@sergey-stratoscale  what will happen otherwise?

**Comment by sergey-stratoscale** [comment]
you'll open .gz files with stratolog if you give generic name to open

**Comment by lior-stratoscale** [comment]
why not read these .gz files instead of ignoring them?

File: py/strato/common/log/log2text.py [hunk 1]
```diff
@@ -244,11 +244,20 @@ def __init__(self, confFile):
             pass
         self.defaultPaths = []
 
-    def findLogFiles(self, providedPaths):
+    def findLogFiles(self, providedPaths, extensionsToIgnore):
         logsToRead = []
         [logsToRead.append(filePath) if os.path.isfile(filePath) else logsToRead.extend(self._tryToMatchShortcut(filePath)) for filePath in providedPaths]
+        self.filterIgnoredExtensions(logsToRead, extensionsToIgnore)
```
Comment (rom-stratoscale) [inline py/strato/common/log/log2text.py]: you ignore the return value

**Comment by sergey-stratoscale** [comment]
@Stratoscale/systegration updated, added with a help from @lior-stratoscale support of .gz file reading

File: py/strato/common/log/log2text.py [hunk 2]
```diff
@@ -236,6 +235,13 @@ def updateConfFile(field, value):
     os.system('sudo mv /tmp/stratolog-conf.tmp %s' % LOG_CONFIG_FILE_PATH)
 
 
+def universalOpen(filePath, mode='r'):
+    if filePath.endswith('.gz'):
```
Comment (rom-stratoscale) [inline py/strato/common/log/log2text.py]: what about gz.1? once there are few rotations.....

**Comment by rom-stratoscale** [comment]
On one hand, you added .gz reader, on other, you added them to ignore list :)

File: py/strato/common/log/log2text.py (hunk 2 above)
Comment (sergey-stratoscale) [inline py/strato/common/log/log2text.py]: extension must come at the end, that's the standard of log rotation

**Comment by sergey-stratoscale** [comment]
In [1]: '.gz'=='tar.gz'
Out[1]: False

**Comment by sergey-stratoscale** [comment]
@Stratoscale/systegration updated

**Pull request merged**